static/script-tests/tests/devices/mediaplayer/live/seekable.js

```diff
@@ -89,7 +127,28 @@
 
     this.LivePlayerSupportLevelSeekableTest.prototype.testSeekableLivePlayerPlayFromCallsFunctionInMediaPlayer = testFunctionsInLivePlayerCallMediaPlayerFunctions('playFrom', 1);
 
-    this.LivePlayerSupportLevelSeekableTest.prototype.testSeekableLivePlayerPauseCallsFunctionInMediaPlayer = testFunctionsInLivePlayerCallMediaPlayerFunctions('pause', 0);
+    this.LivePlayerSupportLevelSeekableTest.prototype.testSeekableLivePlayerPauseCallsFunctionInMediaPlayer = function (queue) {
+        expectAsserts(2);
+        queuedApplicationInit(queue, 'lib/mockapplication', ["antie/devices/mediaplayer/mediaplayer", "antie/devices/device", "antie/devices/mediaplayer/live/seekable"], function (application, MediaPlayer, Device) {
+            var device = new Device(antie.framework.deviceConfiguration);
+            var livePlayer = device.getLivePlayer();
+            var expectedRange = {
+                "start": 0,
+                "end": 30
+            };
+            this.sandbox.stub(livePlayer._mediaPlayer, 'getSeekableRange').returns(expectedRange);
+            this.sandbox.stub(livePlayer._mediaPlayer, 'getCurrentTime').returns(30);
+            this.sandbox.stub(livePlayer._mediaPlayer, 'resume');
+            var mediaPlayerFuncStub = this.sandbox.stub();
+            device.getMediaPlayer().pause = mediaPlayerFuncStub;
```

Comment:
These two lines should probably instead be: `var pauseStub = this.sandbox.stub(device.getMediaPlayer(), 'pause);`

My main problem is that I don't like 'mediaPlayFuncStub' as a name, and would prefer something more descriptive.